core/src/main/java/hudson/slaves/ChannelPinger.java

```diff
@@ -195,6 +195,14 @@ protected void onDead(Throwable cause) {
                         LOGGER.log(Level.FINE, "Ping failed after the channel " + channel.getName() + " is already partially closed.", cause);
                     } else {
                         LOGGER.log(Level.INFO, "Ping failed. Terminating the channel " + channel.getName() + ".", cause);
+                        if (computer == null) {
+                            // Disconnect from agent side.
```

Comment:
From https://github.com/jenkinsci/jenkins/pull/7580. Normally we wait for a fix to be in weeklies for a little while before proposing a backport, right?

Reply:
The RC is scheduled for next week's Wednesday, don't you think 8 days is enough to test this in a weekly?

Reply:
No particular opinion (and as author of the change I would be biased anyway). I recall there being some policy we follow generally, and this does not seem to be an exceptional case—something that has been broken for years, not a recent regression.

Reply:
> something that has been broken for years, not a recent regression.

I see, I missed the 'causes' part on the Jira issue. 2016-era isn't recent, indeed. I dropped the commit.